<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlapBatchSetDialogOptionWidget</class>
 <widget class="QWidget" name="OverlapBatchSetDialogOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="setStartWithCheckBox">
       <property name="text">
        <string>开头为以下内容之一时，将Ovl设置为</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="setStartWithSpinBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>11</number>
         </property>
         <property name="rightMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QRadioButton" name="setStartWithPresetRadioButton">
           <property name="text">
            <string>加载预设</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="setStartWithPresetComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="setStartWithLoadFromFileRadioButton">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从文件加载要匹配的“开头”字符串列表。&lt;/p&gt;&lt;p&gt;文件编码需要是UTF-8。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>从文件加载</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setStartWithBrowseButton">
           <property name="text">
            <string>浏览</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="setStartWithInputRadioButton">
           <property name="text">
            <string>自定义</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setStartWithInputButton">
           <property name="text">
            <string>输入</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="setStartWithListView"/>
      </item>
      <item>
       <widget class="QCheckBox" name="matchStartOtoCheckBox">
        <property name="text">
         <string>同时也匹配开头是“- ”+上述内容的别名</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="OneThirdCheckBox">
     <property name="text">
      <string>将上述操作影响范围之外的Ovl三分之一化</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
